UPGRADE-6.3.md

```diff
@@ -103,3 +103,80 @@ None.
 ## Database Fields
 
 None.
+
+## Backend Theme Library
+
+The Backend was upgraded to Bootstrap 4.1.3
+
+See the [Bootstrap Migration Guide](https://getbootstrap.com/docs/4.1/migration/) for needed changes to your backend modules.
+
+During the upgrade to Bootstrap 4 the following styles where checked and these are our findings:
+
+not found anywhere (so you may skip this search, too):
```

Comment:
Could we move this list to the end of the section? This would make more clear where the list ends, as the other lists don't have an introduction text.

Reply:
you are right. Done.